src/tidy.zig

```diff
@@ -39,7 +39,7 @@ test "tidy" {
     // to read the files once.
     for (paths) |path| {
         const bytes_read = (try std.fs.cwd().readFile(path, buffer)).len;
-        if (bytes_read == buffer.len - 1) return error.FileTooLong;
+        if (bytes_read >= buffer.len - 1) return error.FileTooLong;
```

Comment:
I guess the -1 is to accommodate the \n char, isn't it?

Otherwise all this time we were enforcing 99 characters instead of 100 😱😳

Reply:
Oh! its not the LineTooLong error👍 makes sense now